nemo/collections/llm/gpt/model/__init__.py

```diff
@@ -53,7 +53,7 @@
     LlamaConfig,
     LlamaModel,
 )
-from nemo.collections.llm.gpt.model.mistral import MistralConfig7B, MistralModel
+from nemo.collections.llm.gpt.model.mistral import MistralConfig7B, MistralModel, MistralNeMo2407Config12B
```

Comment:
is 2407 needed in the name? Can't we just use `MistralNeMoConfig12B`?

Reply:
There's two names I think; https://huggingface.co/nvidia/Mistral-NeMo-12B-Base/tree/main but this only contains the nemo checkpoint and https://huggingface.co/mistralai/Mistral-Nemo-Base-2407 which contains the HF checkpoint.